geopandas/geoseries.py

```diff
@@ -1189,12 +1189,18 @@ def estimate_utm_crs(self, datum_name: str = "WGS 84"):
         """
         return self.values.estimate_utm_crs(datum_name)
 
-    def to_json(self, **kwargs) -> str:
+    def to_json(self, show_bbox=True, drop_id=False, **kwargs) -> str:
```

Comment:
was their any reason in your mind why not to also expose the `to_wgs84`? It seems if we are adding this we may as well cover all keywords.

Reply:
No good reason, should have been included upon review, thanks